robocop/checkers/naming.py

```diff
@@ -255,6 +255,13 @@
         Since those words are used for BDD style it's also recommended not to use them within the keyword name.
         """,
     ),
+    "0319": Rule(
+        rule_id="0319",
+        name="deprecated-statement",
+        msg="This keyword '{{ keyword_name }}' is deprecated,"
+            " since version '{{ version }}.*', use {{ alternative }} instead",
+        severity=RuleSeverity.WARNING,
```

Comment:
The min required version should be 5.0 since it's when those keywords started becoming truly deprecate

Reply:
done